.rubocop.yml

```diff
@@ -1,3 +1,154 @@
-inherit_from:
-- https://raw.githubusercontent.com/ManageIQ/guides/master/.rubocop_base.yml
-- .rubocop_local.yml
+# This file is based on https://github.com/rails/rails/blob/master/.rubocop.yml (MIT license)
```

Comment:
This file is not going to play nice with the bot.  Can you add the exclusions to rubocop.local.yml, and then mark this file as not to be touched in the autogenerator?